webapp/components/test-file-results.js

```diff
@@ -194,7 +194,7 @@ class TestFileResults extends WPTFlags(LoadingState(PathInfo(
 
     // Set name for test-level status entry after subtests discovered.
     // Parameter is number of subtests.
-    resultsTable[0].name = this.statusName(resultsTable.length - 1);
+    resultsTable[0].name = this.statusName(resultsTable.length - 2);
```

Comment:
Why is this change made?

Reply:
This is a separate small change that addresses [an old labeling issue](https://github.com/web-platform-tests/wpt.fyi/issues/1958).